linalg_ops/convolution/generate_test_mlir_files.sh

```diff
@@ -76,3 +76,38 @@ for type_combination in ${type_combinations[@]}; do
       --shapes=${shape}
   done
 done
+
+shapes=(
+  "large"
+)
```

Comment:
Any particular reason to only test large shapes with those types and layouts?

Reply:
Yeah, I think we should test more sizes if possible.